bin/hubot

```diff
@@ -1,168 +1,9 @@
 #!/usr/bin/env coffee
-# vim:ft=coffee ts=2 sw=2 et :
-# -*- mode:coffee -*-
 
-Hubot    = require '..'
+# While all other files have been converted to JavaScript via https://github.com/github/hubot/pull/1347,
+# we left the `bin/hubot` file to remain in CoffeeScript in order prevent
+# breaking existing 3rd party adapters of which some are still written in
+# CoffeeScript themselves. We will depracate and eventually remove this file
+# in a future version of hubot
```

Comment:
I'm not sure I understand this.  As long as it still includes a `require('coffee-script')`, shouldn't it be able to load JavaScript?

Also, should we print a deprecation warning that coffee script will not be supported in the major release that follows this one?